src/main/java/mekanism/client/render/item/RenderEnergyCubeItem.java

```diff
@@ -28,50 +32,52 @@
 
     @Override
     protected void renderBlockSpecific(@Nonnull ItemStack stack, TransformType transformType) {
-        GlStateManager.pushMatrix();
         EnergyCubeTier tier = EnergyCubeTier.values()[((ITierItem) stack.getItem()).getBaseTier(stack).ordinal()];
+        GlStateManager.pushMatrix();
+        GlStateManager.rotate(180, 0, 0, 1);
+        GlStateManager.rotate(180, 0, 1, 0);
```

Comment:
If I remember correctly this is unfortunately not the case. I believe I tried it, as well as googled to try and figure out how to combine multiple rotate calls and was unable to find a formula. All I found was people saying that because of how rotations work it is not actually equal, (I think because it turns it 180 degrees along that diagonal or something). I will give it a try before marking this conversation as resolved though.

Reply:
Unsure actually.. though from my testing with the EnegyCubeItem neither are needed as the cube is symmetrical.